tests/Fixtures/app/console

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
```

Comment:
could be placed at the root or under the `bin` folder, it doesn't make much sense to have it so deep in the tree